config/config.go

```diff
@@ -52,7 +56,9 @@ func Get() (*Config, error) {
 		GracefulShutdownTimeout:    5 * time.Second,
 		InputFileAvailableTopic:    "input-file-available",
 		InputFileAvailableGroup:    "input-file-available",
+		KafkaAddr:                  []string{"localhost:9092"},
 		KafkaMaxBytes:              "2000000",
+		KafkaVersion:               "1.0.2",
```

Comment:
Do you need to add the other Kafka fields here? Or are they intentionally excluded because they're not needed/used yet?

Reply:
Those fields are used when `Kafka TLS` is enabled otherwise they are not used